apiServer.js

```diff
@@ -143,7 +156,7 @@ app.get('/libraries/:library', function (req, res) {
     return false;
   });
 
-  if (fields && results.length > 0) {
+  if (fields.length && results.length) {
```

Comment:
`> 0` is more readable and shouldn't be removed in this PR

Reply:
I'll create a new PR to add `> 0` in for both then.